src/components/input/demo/index.tsx

```diff
@@ -12,6 +12,10 @@ export class SmoothlyInputDemo {
 		console.log("selectionChanged", event.detail)
 	}
 
+	testSubmit(e: CustomEvent) {
+		console.log(e.detail)
+	}
+
```

Comment:
last comment marked resolved without comment or doing what my comment was about. is this intended?